### PR TITLE
Display discord bridged posts #19233

### LINKED ISSUE #19233: Display discord bridged posts
As a user
I want to see posts bridged from discord
So that I can see posts from users not on status

### Description

Currently it's possible to bridge a community with discord.
Posts posted from discord, are likely not showed properly on status. 
The easiest way is to get access to the community, @pavloburykh should be able to help, provided that access has been granted to the test user.
This is how roughly it should look like (use mobile components),UI is not critical, as long as it looks roughly ok:

image

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #19233

image

### CODE REVIEW

File: src/status_im/contexts/chat/messenger/messages/content/view.cljs [hunk 1]
```diff
@@ -113,6 +114,29 @@
...
+     [fast-image/fast-image
+      {:source {:uri user-avatar}
+       :style  {:width         32
+                :margin-top    4
+                :border-radius 16
+                :height        32}}]
+     [rn/view {:margin-left 8 :flex 1}
+      [quo/author
+       {:primary-name   (str user-name)
+        :short-chat-key (str "Bridged from " bridge-name)
```
Comment (cammellos) [inline src/status_im/contexts/chat/messenger/messages/content/view.cljs:139]: maybe you want a translation key here? not the end of the world, but just for consistency

**Review by cammellos — APPROVED** [review]

File: src/status_im/contexts/chat/messenger/messages/content/view.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/contexts/chat/messenger/messages/content/view.cljs:139]: we're not sure if we need this, we don't have designs I guess yet?

Comment (flexsurfer) [reply src/status_im/contexts/chat/messenger/messages/content/view.cljs:139]: so the entire implementation is kinda of temporary?

Comment (cammellos) [reply src/status_im/contexts/chat/messenger/messages/content/view.cljs:139]: oh yes, I guess so, though we are pretty certain `Briged from discord` will be there, but no strong opinion on my side, worse things happen at sea :)

File: src/status_im/contexts/chat/messenger/messages/content/view.cljs [hunk 2]
```diff
@@ -113,6 +115,34 @@
       constants/content-type-system-message-mutual-event-sent
       [system-message-contact-request message-data :contact-request])))
 
+(defn bridge-message-content
+  [{:keys [bridge-message timestamp]}]
+  (let [{:keys [user-avatar user-name
+                bridge-name content]} bridge-message
+        user-name                     (when (string? user-name)
```
Comment (BalogunofAfrica) [inline src/status_im/contexts/chat/messenger/messages/content/view.cljs:122]: I think there is a conflict between the name of the destructed key `user-name` and the variable created on line 122. Perhaps a distinction between the two?

**Comment by pavloburykh** [comment]
@flexsurfer thank you a lot for the PR! 

If the primary acceptance criteria is ```bridged text messages should be displayed on mobile``` then PR is ready for merge. 

Mind, that there are plenty of uncovered scenarios, such as: displaying other types of data (i.e images/formatted text/gifs), replies, reactions, implementing interactions with bridged messages, deleting messages etc but it is out of scope of this PR so I have ignored them while testing.  cc @cammellos

File: src/status_im/contexts/chat/messenger/messages/content/view.cljs [hunk 3]
```diff
@@ -113,6 +115,34 @@
...
+                :border-radius 16
+                :height        32}}]
+     [rn/view {:margin-left 8 :flex 1}
+      [quo/author
+       {:primary-name   (str user-name)
+        :short-chat-key (str "Bridged from " bridge-name)
+        :time-str       (datetime/timestamp->time timestamp)}]
+      [quo/text
+       {:size  :paragraph-1
+        :style {:line-height 22.75}}
```
Comment (BalogunofAfrica) [inline src/status_im/contexts/chat/messenger/messages/content/view.cljs:143]: Shouldn't there be predefined size for this particular `lineHeight`?

File: src/status_im/contexts/chat/messenger/messages/content/view.cljs (hunk 2 above)
Comment (flexsurfer) [reply src/status_im/contexts/chat/messenger/messages/content/view.cljs:122]: no conflicts , it's intentional

File: src/status_im/contexts/chat/messenger/messages/content/view.cljs (hunk 3 above)
Comment (flexsurfer) [reply src/status_im/contexts/chat/messenger/messages/content/view.cljs:143]: i took it from message component, so probably if we want to change this should be done in message as well, but this code is temporary implementation

**Pull request merged**